.circleci/config.yml

```diff
@@ -69,7 +68,7 @@ jobs:
     - store_artifacts:
         path: tests/backend/results
     - save_cache:
-        key: cached-dependencies
+        key: node-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

Comment:
replace that line by `key: yarn-packages-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}`